cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/aggregation/NGSIGenericRowAggregator.java

```diff
@@ -92,7 +96,8 @@ public void aggregate(NGSIEvent event) {
             aggregation.get(NGSIConstants.ATTR_NAME).add(new JsonPrimitive(attrName));
             aggregation.get(NGSIConstants.ATTR_TYPE).add(new JsonPrimitive(attrType));
             aggregation.get(NGSIConstants.ATTR_VALUE).add(attrValue);
-            aggregation.get(NGSIConstants.ATTR_MD).add(new JsonPrimitive(attrMetadata));
+            aggregation.get(NGSIConstants.ATTR_MD).add(jsonAttrMetadata);
+            //aggregation.get(NGSIConstants.ATTR_MD).add(new JsonPrimitive(attrMetadata));
```

Comment:
comment legacy?

Reply:
Fixed in [611a63c](https://github.com/telefonicaid/fiware-cygnus/pull/1923/commits/611a63ca19c1bcebc88e8d2bca4119031679c8b0)